olp-cpp-sdk-dataservice-read/tests/VolatileLayerClientImplTest.cpp

```diff
@@ -554,8 +548,7 @@ TEST(VolatileLayerClientImplTest, PrefetchTiles) {
 
     ASSERT_NE(future.wait_for(kTimeout), std::future_status::timeout);
     read::PrefetchTilesResponse response = future.get();
-    ASSERT_TRUE(response.IsSuccessful()) << response.GetError().GetMessage();
-    ASSERT_TRUE(response.GetResult().empty());
+    ASSERT_FALSE(response.IsSuccessful()) << response.GetError().GetMessage();
   }
   {
     SCOPED_TRACE("Levels not specified.");
```

Comment:
Are there any difference between "Prefetch tiles with default levels" and "Levels not specified." tests? Looks identical for me.